tests/ImageSharp.Tests/TestUtilities/TestEnvironment.Formats.cs

```diff
@@ -60,11 +60,10 @@ private static Configuration CreateDefaultConfiguration()
 
             if (!IsLinux)
             {
-                configuration.ConfigureCodecs(
-                    ImageFormats.Png,
-                    SystemDrawingReferenceDecoder.Instance,
-                    SystemDrawingReferenceEncoder.Png,
-                    new PngImageFormatDetector());
+                // System.Drawing on Windows can decode 48bit and 64bit pngs but
+                // it doesn't preserve the accuracy we require for comparison.
+                // This makes CompareToOriginal method non-useful.
+                configuration.Configure(new PngConfigurationModule());
```

Comment:
We should find a different testing method for those bit depths then, otherwise we are testing `PngDecoder` against itself in all our test cases, loosing the point of the regression testing.

Reply:
Yup, very much agree.